plugins/xcode/xcode

```diff
@@ -3,6 +3,18 @@
 export DOTFILES_XCODE_DEPS=('homebrew_core')
 
 dotfiles_xcode_apply () {
+  if xcode-select -p 1>/dev/null; then
+    debug 'Xcode is already installed, skipping it.'
+  else
+    _install_xcode_from_mas
+  fi
+
+  debug 'Making sure we have accepted the xcode license'
+  sudo xcodebuild -license accept
+}
+
+_install_xcode_from_mas() {
+  mv XcodeBrewfile Brewfile
```

Comment:
This needs to be updated to not be relative

Reply:
But what's it trying to do?

Reply:
oh, i see. I completely missed the file rename above.

Reply:
Yeah, I'm open to other ides for how to make it skip this Brewfile if you prefer.

Reply:
nah, this makes sense, I just missed the rename during the code review, so I was very confused.